portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -7,17 +7,13 @@
  public class Comment {
     private String sender;
     private String text;
+    private int id;
```

Comment:
Nit: While it doesn't apply here, as a rule of thumb, ids should always be longs. (Ask me about the time I got called at 2am because someone made this mistake.)

On a more practical note, you don't seem to use this id anywhere, so maybe it should be removed.

Reply:
Will do! Must've forgotten to take it out.